src/bric.c

```diff
@@ -1,8 +1,8 @@
 #include "bric.h"
 
 static struct __current_file CurrentFile;
-static struct editor_config Editor;
-static struct termios orig_termios; // so we can restore original at exit
```

Comment:
We need these files to global, so we could access them in some functions who don't accept parameters, like `editor_at_exit()`.